src/sentry/discover/translation/mep_to_eap.py

```diff
@@ -14,6 +15,34 @@ class QueryParts(TypedDict):
     orderby: list[str] | None
 
 
+PERCENTILE_REGEX = r"percentile\(.*\)"
```

Comment:
nit: would `startswith("percentile(")` work just as fine since `function_switcheroo` is already in the aggregate visitor?